src/status_im/chat/events/commands.cljs

```diff
@@ -70,7 +70,6 @@
   (fn [_ [{command-name :name}]]
     (case (keyword command-name)
       :grant-permissions
-      {:dispatch [:request-permissions
-                  [:read-external-storage]
-                  #(re-frame/dispatch [:initialize-geth])]}
+      {:dispatch [:request-permissions {:permissions [:read-external-storage]
+                                        :on-allowed  #(re-frame/dispatch [:initialize-geth])}]}
```

Comment:
Just a remark, `on-denied` not supplied here. I wonder who requests permissions in this case and what is he supposed to see if user denies.

Reply:
By default nothing happens if user accepts or denies permissions so nothing
will happen. But I think this permission is supposed to be accepted when
user installs so it won't even ask. I kept the old behavior but we can add
an alert and close since geth can't work without it.